src/app/routes/error/index.js

```diff
@@ -5,6 +5,11 @@ export default {
   path: errorPagePath,
   exact: true,
   component: ErrorPage,
-  getInitialData: () => Promise.resolve({ status: 200 }),
+  getInitialData: pathname => {
+    const [, , errorPage] = pathname.split('/');
```

Comment:
This array destructoring looks as though it's a mistake, instead of a deliberate task of getting the string '404'/'500' from the pathname. 
Is it possible to get the error code string via React router methods instead, rather than us extracting it manually here? This would make this code less brittle, so that if the regex updates, this line won't break the Error page implementation.

Reply:
You're right this is much better https://github.com/bbc/simorgh/pull/5714/commits/d008b252514689501a78501f65cf64279311dedd